dev-support/rat/rat-exclusions.txt

```diff
@@ -0,0 +1,55 @@
+###### Licensed to the Apache Software Foundation (ASF) under one
+###### or more contributor license agreements.  See the NOTICE file
+###### distributed with this work for additional information
+###### regarding copyright ownership.  The ASF licenses this file
+###### to you under the Apache License, Version 2.0 (the
+###### "License"); you may not use this file except in compliance
+###### with the License.  You may obtain a copy of the License at
+######
+######   http://www.apache.org/licenses/LICENSE-2.0
+######
+###### Unless required by applicable law or agreed to in writing,
+###### software distributed under the License is distributed on an
+###### "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+###### KIND, either express or implied.  See the License for the
+###### specific language governing permissions and limitations
+###### under the License.
+
+.github/*
+CONTRIBUTING.md
+README.md
+SECURITY.md
+.gitattributes
+src/main/resources/proto.lock
+**/output.xml
+**/log.html
+**/report.html
+**/.ssh/id_rsa*
+src/test/resources/*.log
+src/test/resources/ssl/*
+**/dependency-reduced-pom.xml
+**/pnpm-lock.yaml
+src/test/resources/prometheus-test-response.txt
+src/main/license/**
+src/main/resources/proto.lock
+src/test/resources/test.db.ini
+tools/fault-injection-service/README.md
+**/webapps/static/angular-nvd3-1.0.9.min.js
+**/webapps/static/angular-route-1.8.0.min.js
+**/webapps/static/bootstrap-3.4.1/**
+**/webapps/static/d3-3.5.17.min.js
+**/webapps/static/jquery-3.5.1.min.js
+**/webapps/static/nvd3-1.8.5.min.css.map
+**/webapps/static/nvd3-1.8.5.min.css
+**/webapps/static/nvd3-1.8.5.min.js.map
+**/webapps/static/nvd3-1.8.5.min.js
+**/webapps/static/angular-1.8.0.min.js
+src/test/resources/additionalfields.container
+src/test/resources/incorrect.checksum.container
+src/test/resources/incorrect.container
+src/test/resources/test.db.ini
+src/test/resources/123-dn-container.db/**
+src/test/resources/123.container
+static/slides/*
+**/themes/ozonedoc/**
+**/*.json
```

Comment:
It would be nice to sort these.

To make the script useful for local run by devs, we should also exclude:

```
dev-support/ci/bats-assert/**
dev-support/ci/bats-support/**
.dev-tools/**
```

(Would be useful if we could exclude everything in `.gitignore` by default...)